examples/src/test/resources/examples/osm_custom_params/oldApi-requestData.json

```diff
@@ -0,0 +1,67 @@
+{
+  "app" : "mapbox xyz old api",
+  "srs": "EPSG:3857",
+  "layout": "A4 landscape",
+  "outputFilename": "exemplemapbox",
+  "dpi": 72,
+  "outputFormat": "pdf",
+  "layers": [
+    {
+      "baseURL": "https://api.tiles.mapbox.com/v4/tmcw.map-7s15q36b/",
```

Comment:
I don't care for tests that require external resources.  For example the server could crash or be removed.  Could you do like I did for OSM?  I downloaded the tiles that are required for the example and put them in:  `examples/src/main/webapp/data/www/osm` 

You can do the same and put them in   `examples/src/main/webapp/data/www/mapbox` then change baseURL to be: `"http://localhost:9876/e2egeoserver/www/mapbox"`

See the OSM example: `examples/src/test/resources/examples/print_osm_new_york_EPSG_900913/requestData-scale-osm.json`